frame/staking/src/mock.rs

```diff
@@ -990,19 +997,19 @@ pub(crate) fn make_all_reward_payment(era: EraIndex) {
 macro_rules! assert_session_era {
 	($session:expr, $era:expr) => {
 		assert_eq!(
-			Session::current_index(),
-			$session,
-			"wrong session {} != {}",
-			Session::current_index(),
-			$session,
-		);
-		assert_eq!(
-			Staking::active_era().unwrap().index,
-			$era,
-			"wrong active era {} != {}",
-			Staking::active_era().unwrap().index,
-			$era,
-		);
+															Session::current_index(),
```

Comment:
:-|

Reply:
Okay this is where rustfmt is being wonky :\ I am not sure why but sometimes with macros it does this where it infinitely pushes them forward. 

Apologies for the occasional styling blimps, I am giving rustfmt in a contained manner (only format code that I touched + limited configs) a try.